recipes/enum34/meta.yaml

```diff
@@ -0,0 +1,36 @@
+{% set name = "enum34" %}
+{% set version = "1.1.6" %}
+{% set sha256 = "8ad8c4783bf61ded74527bffb48ed9b54166685e4230386a9ed9b1279e2df5b1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
```

Comment:
Appears I forgot as `skip`. :facepalm: Please see PR ( https://github.com/conda-forge/enum34-feedstock/pull/1 ) for the fix.